### DISCUSSION TITLE
Any data fetching library for Svelte like `tanstack/query` or `swr`?

### DISCUSSION THREAD

**Discussion by frederikhors** [sveltejs/kit#10175] [Q&A]
I would like to know if there is anyone among you who is using [@tanstack/query](https://github.com/TanStack/query) and if they like it.

I'm thinking of converting a large SAAS application from React to Svelte.

The first snag is that with `TanStack/query` [there isn't yet a way to persist data in the browser (asynchronously with IndexedDB)](https://github.com/TanStack/query/discussions/5463).

What do you recommend?

Are there any other data fetching libraries in Svelte? Something like `tanStack/query` or [swr](https://github.com/vercel/swr)?

**Comment by ghost** [comment]
there's one more library [hyperfetch](https://hyperfetch.bettertyped.com/), that supports caching, along with queues for mutations. You could give it a try.

**Comment by o-az** [comment]
@frederikhors what did you end up using?
